src/Main.java

```diff
@@ -0,0 +1,10 @@
+import java.io.File;
+
+public class Main {
+
+    public static void main(String[] args) {
+        SearchEngine searchEngine = new SearchEngine();
+        File file = new File(args[1]);
+        searchEngine.run(file);
```

Comment:
The file shouldn't be created here and passed into `SearchEngine`. Can you create a new class where you will implement the `readFile` operation?

Reply:
created new FileManager class

Reply:
`ReadManager` is a pretty abstract name and doesn't say what it does. It keeps only one method that reads from the file, so `FileReader` I think should match better here.

Reply:
renamed